docs/source/kit/arduino.rst

```diff
@@ -0,0 +1,63 @@
+Arduino
+=======
+
+This board allows you to control GPIO pins and analogue pins. More specifically, it's an `Arduino Uno <https://store.arduino.cc/arduino-uno-rev3>`__.
+
+.. figure:: /_static/kit/arduino_headers.png
+   :alt: Arduino
+   :scale: 75%
+
+   Arduino Uno
+
+Headers
+-------
+
+We have supplied 2 screw terminal headers for your Arduino, allowing you to easily, and securely attach your sensors.
+
+The reset button
+----------------
+
+The reset button allows you to instantly reboot the Arduino in case it
+isn’t working. This is not a guaranteed fix, but may solve some
+problems.
+
+GPIO Pins
+---------
+
+The Arduino allows you to connect your kit to your own electronics. It has fourteen digital I/O pins, and six analogue. The analogue pins can read an analogue signal from 0 to 5V. The board also has a couple of ground pins, as well as some pins fixed at 3.3V and 5V output.
+
+The pin layout of the servo assembly is the same as an Arduino Uno.
```

Comment:
The servo assembly isn't relevant anymore. I don't think this line is needed at all now.